packages/libraries/core/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * from '@fluentui-react-native/button';
 export * from '@fluentui-react-native/callout';
 export * from '@fluentui-react-native/checkbox';
 export * from '@fluentui-react-native/contextual-menu';
+export * from '@fluentui-react-native/experimental-avatar';
```

Comment:
Do we want to put it in the library when it is still experimental? Users can just pull it in directly. I would expect that we would want to figure out the intersection between Avatar and Persona before it goes in the library.

Reply:
Good point. If users can pull it in separately, then that may actually be better for getting this used by our first client, as they won't need to update their version of FURN just yet. Thanks for the recommendation =)